frontend/packages/modelServing/src/components/projectDetails/DeploymentsTable.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {
+  SortableData,
+  Table,
+  TableRowTitleDescription,
+} from '@odh-dashboard/internal/components/table/index';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import ResourceTr from '@odh-dashboard/internal/components/ResourceTr';
+import { ActionsColumn, Td } from '@patternfly/react-table';
+import { useResolvedExtensions } from '@odh-dashboard/plugin-core';
+import { ModelServingPlatform } from '../../concepts/modelServingPlatforms';
+import { Deployment, isModelServingDeploymentsTableExtension } from '../../../extension-points';
+
+const genericColumns: SortableData<Deployment>[] = [
```

Comment:
Might be worth adding inference endpoint? I feel like we can expect every deployment to have this